crux/models/resource.py

```diff
@@ -181,31 +170,20 @@ def folder(self):
         self._folder = self._get_folder()
         return self._folder
 
+    @folder.setter
+    def folder(self, folder):
+        self.object_model["folder"] = folder
+        log.debug(self.object_model)
+        self._folder = folder
```

Comment:
Is it possible to set the folder location by PUTing an update with a new folder path? Would about with updating the `folderId`?

Something is weird about the folder behavior in the API if it provides a `null` folder field.